### ISSUE TITLE
nodejs process stays alive for another 20 seconds after disconnect

### ISSUE THREAD

**Issue by michielbdejong** [ripple/ripple-lib#792]
To reproduce:

```js
const RippleAPI = require('ripple-lib').RippleAPI
const api = new RippleAPI({ server: 'wss://s.altnet.rippletest.net:51233' })
api.connect().then(() => {
  console.log('connected')
  return api.disconnect()
}).then(() => {
  console.log('disconnected')
})
```

And then:
```sh
$ time node test-disconnect.js
connected
disconnected

real	0m20.853s
user	0m0.687s
sys	0m0.061s
```

Continuation of https://github.com/ripple/ilp-plugin-xrp-escrow/issues/14

**Comment by michielbdejong** [comment]
@sentientwaffle I think you fixed what sounds like the same issue in https://github.com/interledgerjs/ilp-plugin-bells/pull/153 right?

**Comment by sentientwaffle** [comment]
It does look similar. I think `unref()`ing the timer on https://github.com/ripple/ripple-lib/blob/a9e5ac6290ef2fa17fadb34e808e4f5080d7ba32/src/common/connection.js#L443 will fix the issue.

**Comment by FKSRipple** [comment]
Fixed by #773
